modules/fundamental/src/purl/endpoints/mod.rs

```diff
@@ -81,5 +82,21 @@ pub async fn all(
     Ok(HttpResponse::Ok().json(service.purls(search, paginated, db.as_ref()).await?))
 }
 
+#[utoipa::path(
+    operation_id = "garbageCollect",
+    tag = "purl",
+    responses(
+        (status = 200, description = "Performs garbage collection for orphaned packages", body = String),
+    ),
+)]
+#[get("/v2/purl/gc")]
+pub async fn gc(
+    service: web::Data<PurlService>,
+    db: web::Data<Database>,
+    _: Require<ReadSbom>,
```

Comment:
To stay consistent with how it previously worked, this should probably require delete privs.
```suggestion
    _: Require<DeleteSbom>,
```

Reply:
thanks, pr updated :+1: